{
  "configurations": [
    {
      "nodes": [
        {
          "cpeMatch": [
            {
              "criteria": "cpe:2.3:a:wisc:htcondor:9.0.0:*:*:*:*:*:*:*",
              "matchCriteriaId": "8252CB70-D65B-41CE-96C4-A8C5D048C43A",
              "vulnerable": true
            },
            {
              "criteria": "cpe:2.3:a:wisc:htcondor:9.0.1:*:*:*:*:*:*:*",
              "matchCriteriaId": "016130CA-A4D7-4069-BD8A-DED4C51966EA",
              "vulnerable": true
            },
            {
              "criteria": "cpe:2.3:a:wisc:htcondor:9.0.2:*:*:*:*:*:*:*",
              "matchCriteriaId": "02D6E3E0-B171-4E0A-A5B6-A13FE2294805",
              "vulnerable": true
            },
            {
              "criteria": "cpe:2.3:a:wisc:htcondor:9.1.0:*:*:*:*:*:*:*",
              "matchCriteriaId": "E02EB126-A42E-4C90-966D-8A54AE50230F",
              "vulnerable": true
            }
          ],
          "negate": false,
          "operator": "OR"
        }
      ]
    }
  ],
  "cveTags": [],
  "descriptions": [
    {
      "lang": "en",
      "value": "An issue was discovered in HTCondor 9.0.x before 9.0.4 and 9.1.x before 9.1.2. When authenticating to an HTCondor daemon using a SciToken, a user may be granted authorizations beyond what the token should allow."
    },
    {
      "lang": "es",
      "value": "Se ha detectado un problema en HTCondor versiones 9.0.x anteriores a 9.0.4 y versiones 9.1.x anteriores a 9.1.2. Cuando es autenticado un demonio HTCondor usando un SciToken, un usuario puede recibir autorizaciones más allá de lo que el token debería permitir"
    }
  ],
  "id": "CVE-2021-45102",
  "lastModified": "2021-12-22T19:51:33.407",
  "metrics": {
    "cvssMetricV2": [
      {
        "acInsufInfo": false,
        "baseSeverity": "MEDIUM",
        "cvssData": {
          "accessComplexity": "LOW",
          "accessVector": "NETWORK",
          "authentication": "SINGLE",
          "availabilityImpact": "PARTIAL",
          "baseScore": 6.5,
          "confidentialityImpact": "PARTIAL",
          "integrityImpact": "PARTIAL",
          "vectorString": "AV:N/AC:L/Au:S/C:P/I:P/A:P",
          "version": "2.0"
        },
        "exploitabilityScore": 8.0,
        "impactScore": 6.4,
        "obtainAllPrivilege": false,
        "obtainOtherPrivilege": false,
        "obtainUserPrivilege": false,
        "source": "nvd@nist.gov",
        "type": "Primary",
        "userInteractionRequired": false
      }
    ],
    "cvssMetricV31": [
      {
        "cvssData": {
          "attackComplexity": "LOW",
          "attackVector": "NETWORK",
          "availabilityImpact": "HIGH",
          "baseScore": 8.8,
          "baseSeverity": "HIGH",
          "confidentialityImpact": "HIGH",
          "integrityImpact": "HIGH",
          "privilegesRequired": "LOW",
          "scope": "UNCHANGED",
          "userInteraction": "NONE",
          "vectorString": "CVSS:3.1/AV:N/AC:L/PR:L/UI:N/S:U/C:H/I:H/A:H",
          "version": "3.1"
        },
        "exploitabilityScore": 2.8,
        "impactScore": 5.9,
        "source": "nvd@nist.gov",
        "type": "Primary"
      }
    ]
  },
  "published": "2021-12-16T05:15:08.917",
  "references": [
    {
      "source": "cve@mitre.org",
      "tags": [
        "Mitigation",
        "Vendor Advisory"
      ],
      "url": "https://research.cs.wisc.edu/htcondor/security/vulnerabilities/HTCONDOR-2021-0004/"
    }
  ],
  "sourceIdentifier": "cve@mitre.org",
  "vulnStatus": "Analyzed",
  "weaknesses": [
    {
      "description": [
        {
          "lang": "en",
          "value": "CWE-863"
        }
      ],
      "source": "nvd@nist.gov",
      "type": "Primary"
    }
  ]
}